x-pack/plugins/observability_solution/dataset_quality/common/constants.ts

```diff
@@ -19,3 +19,9 @@ export const NONE = 'none';
 
 export const DEFAULT_TIME_RANGE = { from: 'now-24h', to: 'now' };
 export const DEFAULT_DEGRADED_DOCS = { percentage: 0, count: 0 };
+
+export const SO_SEARCH_LIMIT = 300;
+export const PACKAGES_SAVED_OBJECT_TYPE = 'epm-packages';
+export const DASHBOARD_SAVED_OBJECT_TYPE = 'dashboard';
+
+export const DASHBOARD_APP_LOCATOR = 'DASHBOARD_APP_LOCATOR';
```

Comment:
Is there a way we can also get this one from a shared location? I've only found it in `src/plugins/dashboard/public/dashboard_app/locator/locator.ts` 😶